pkg/cmd/server/bootstrappolicy/controller_policy.go

```diff
@@ -55,6 +55,8 @@ func init() {
 			rbac.NewRule("get").Groups(imageGroup, legacyImageGroup).Resources("imagestreams").RuleOrDie(),
 			rbac.NewRule("get", "list", "create", "delete").Groups(kapiGroup).Resources("pods").RuleOrDie(),
 			rbac.NewRule("get").Groups(kapiGroup).Resources("namespaces").RuleOrDie(),
+			rbac.NewRule("get").Groups(buildGroup, legacyBuildGroup).Resources("secrets").RuleOrDie(),
+			rbac.NewRule("get").Groups(buildGroup, legacyBuildGroup).Resources("configmaps").RuleOrDie(),
```

Comment:
@csrwng hmm. I think that with this change, having create/update on builds will practically imply having get on secrets.  Because if there's a secret I can't read, I could now create a build, refer in it to the unreadable secret, and later read the secret contents from the resulting env var, on or in the resulting build pod.

Now, it may be the case that having create/update on builds already practically implies having get on secrets, but if not, this is a new window.